rcl/src/rcl/logging.c

```diff
@@ -115,7 +131,6 @@ rcl_ret_t rcl_logging_fini()
   if (RCL_RET_OK == status && g_rcl_logging_ext_lib_enabled) {
     status = rcl_logging_external_shutdown();
   }
-
```

Comment:
nitpick: drop this change, its not needed

Reply:
done